crates/manager/src/executor/eigen_event_handler.rs

```diff
@@ -0,0 +1,130 @@
+use crate::config::BlueprintManagerContext;
```

Comment:
This entire file seems wrong. There is nothing exactly _unique_ about Eigenlayer events. Eigenlayer is a protocol on EVM that uses Solidity smart contracts or otherwise EVM compatible languages to emit events. The EVM event handling utilities should be sufficient for anything here.